Dockerfile

```diff
@@ -13,15 +13,15 @@ LABEL summary="Automated releasing from GitHub repositories" \
       io.k8s.display-name="Release Bot" \
       io.openshift.tags="builder" \
       io.openshift.s2i.scripts-url="$STI_SCRIPTS_URL" \
-      usage="s2i build <SOURCE-REPOSITORY> koscicz/release-bot <APP-NAME>"
+      usage="s2i build <CONFIGURATION-REPOSITORY> usercont/release-bot <APP-NAME>"
 
-RUN dnf install -y python3 python2 python-pip fedpkg git krb5-workstation
+RUN dnf install -y python2-pip fedpkg git krb5-workstation
```

Comment:
Any reason for removing python3?

Reply:
because it's installed by default on fedora:27
The commit messages explain my steps.